clio-client/src/test/scala/org/broadinstitute/clio/client/dispatch/AddGvcfSpec.scala

```diff
@@ -47,4 +48,21 @@ class AddGvcfSpec extends BaseClientSpec {
       .map(_ shouldBe an[UpsertId])
   }
 
+  it should "fail to add a gvcf that would overwrite an existing document" in {
+    recoverToSucceededIf[Exception] {
+      val mockIoUtil = new MockIoUtil
+
+      succeedingDispatcher(mockIoUtil, testGvcfLocation)
+        .dispatch(goodGvcfAddCommand)
+        .map(_ shouldBe an[UpsertId])
```

Comment:
I'm missing something here... have we already set up the `succeedingDispatcher` to return data for the key in `goodGvcfAddCommand`?

Reply:
I'd also remove the `.map(_ shouldBe ...)`, since it's not an assertion that we expect to run / that we care about for this test.

Reply:
I think the succeeding dispatcher just returns the json that you set.. so it isn't a real key lookup. We could have succeedingDispatcher add the same data and then change it and run it again.. but I think it ends up testing the same thing essentially.